index.js

```diff
@@ -1687,7 +1742,62 @@
   //. //
   //. //   The value at position 1 is not a member of ‘{ x :: FiniteNumber, y :: FiniteNumber }’.
   //. ```
-  function RecordType(fields) {
+  //.
+  //. ##### Extending record types
+  //.
+  //. By using the `parent` argument of the `RecordType` constructor, one can
+  //. define record types which must contain all the same properties of
+  //. another. For example, one could define `Point3D` as a refinement of
+  //. `Point` from the previous example:
+  //.
+  //. ```javascript
+  //. //    Point3D :: Type
+  //. const Point3D = $.RecordType(Point, {z: $.FiniteNumber});
+  //.
+  //. //    p1 :: Point
+  //. const p1 = {x: 0, y: 0};
+  //.
+  //. //    p2 :: Point, Point3D
+  //. const p2 = {x: 3, y: 4, z: 5};
+  //.
+  //. Point.validate(p1);    // => Right({x: 0, y: 0})
+  //. Point3D.validate(p1);  // => Left({propPath: ['z'], value: undefined})
+  //.
+  //. Point.validate(p2);    // => Right({x: 3, y: 4, z: 5})
+  //. Point3D.validate(p2);  // => Right({x: 3, y: 4, z: 5})
+  //.
+  //. dist(p1, p2);
+  //. // => 5
+  //. ```
+  //.
+  //. The `p2` value is a member of `Point3D` as well as `Point`, making it a
+  //. valid argument to `dist`. By default record types allow additional
+  //. properties (which is what allowed `p2` to be a member of `Point`). If one
```

Comment:
I :heart: the concrete clause. In fact, I would lead with it:

_By default, record types permit the presence of additional fields. As a result, `p2` is a member of `Point` as well as `Point3D`._

_One could define `Strict :: Type -> Type` and use it to define record types which do not permit additional fields:_